File: JUNGOL/Beginner/#1304 숫자사각형3/숫자사각형3.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
    int n; cin>>n;
    vector<vector<int>> square;
    for(int i=0; i<n; i++){
        vector<int> v(n);
        square.push_back(v);
    }

    int num = 1;
    for(int i=0; i<n; i++)
        for(int j=0; j<n; j++)
            square[j][i] = num++;

    for(vector<int> s : square){
        for(int si : s)
            printf("%d ", si);
        cout<<endl;
    }
}

File: JUNGOL/Beginner/#1304 숫자사각형3/숫자사각형3.py
n = int(input())
square = [[0 for i in range(n)] for i in range(n)]

num = 1
for i in range(n):
    for j in range(n):
        square[j][i] = num
        num += 1

for s in square:
    for si in s:
        print(si, end=' ')
    print()
